modules/teacher.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'person'
+
+# top level comment on Teacher Class for linter error
+class Teacher < Person
+  attr_accessor :specialization
+
+  def initialize(age, specialization, name = 'Unknown', parent_permission: true)
+    @specialization = specialization
+    super(name, age, parent_permission)
+  end
+
+  def can_use_services
+    true
+  end
```

Comment:
- [ ] This method is intended to override the one in the Person class so that means the names should match. Kindly add "?" to the method name: `can_use_services?`